### ISSUE TITLE
GIT_PYTHON_TRACE to use log.debug instead of log.info

### ISSUE THREAD

**Issue by gsemet** [gitpython-developers/GitPython#792]
Is it possible to change the default behavior of `GIT_PYTHON_TRACE=full` to use log level debug instead of INFO?

It is nice to have this trace on logging, but I would like to be able to route them to a debug.log file while keeping the stdout in INFO loglevel and higher only.

Thanks

**Comment by Byron** [comment]
I agree, by making it debug it probably interacts better with existing application logging, which may already use the `info()` channel for something less verbose than what gitpython would produce.
A patch is very welcome.
